config/crd/bases/dynatrace.com_dynakubes.yaml

```diff
@@ -2290,6 +2290,9 @@ spec:
                         description: Time of the last connection request
```

Comment:
- It works for the `oneAgent`, I guess in the worst case we have to wait a whole reconciliation cycle, don't we? I'm asking because if that *is* the case, can/should we improve? Maybe somehow with conditions?
- for `activeGate` it does not work for me. Does it work for you? It just did not restart ever.

Reply:
In the worst case we have to wait the time till the cache times out (the conditions we define) and then till the next reconciliation. But that's ok in my opinion, as we want to limit the number of requests to the cluster.

ActiveGate does not update due to a cluster bug, a fix is already in the pipeline. Currently it always returns the same token, even after the rotation started, that's the reason why the ActiveGate does not restart.